build.gradle

```diff
@@ -109,6 +109,7 @@ subprojects { project ->
     jvmArgs += "-Dlistener=okhttp3.testing.InstallUncaughtExceptionHandlerListener"
     jvmArgs += "-Dokhttp.platform=$platform"
 
+    maxParallelForks Runtime.runtime.availableProcessors().intdiv(2) ?: 1
```

Comment:
Would love to see this land.  CI env in circle is 2 CPU only, so I think this is no-op.

<img width="187" alt="image" src="https://user-images.githubusercontent.com/231923/56237456-f04bea00-6083-11e9-8c59-cab818e0019f.png">

I'd suggest we try to flush out the flaky tests by running with Runtime.runtime.availableProcessors() * 2, and seeing what breaks.  A lot of tests are heavily CPU bound anyway.  If you enable this before any capacity boost, then it might block running on a bigger paid container.

Reply:
3236112